src/dbally/nl_responder/token_counters.py

```diff
@@ -34,6 +34,33 @@ def count_tokens_for_openai(messages: ChatFormat, fmt: Dict[str, str], model: st
     return num_tokens
 
 
+def count_tokens_for_anthropic(messages: ChatFormat, fmt: Dict[str, str]) -> int:
+    """
+    Counts the number of tokens in the messages for Anthropic's models.
+
+    Args:
+        messages: Messages to count tokens for.
+        fmt: Arguments to be used with prompt.
+
+    Returns:
+        Number of tokens in the messages.
+
+    Raises:
+        ImportError: If anthropic package is not installed.
+    """
+
+    try:
+        from anthropic._tokenizers import sync_get_tokenizer  # pylint: disable=import-outside-toplevel
```

Comment:
Is there async version? Maybe we should use it instead

Reply:
Yes, there is. Good idea, will change it.